teams/teams-ps/teams/Update-CsTeamTemplate.md

```diff
@@ -123,6 +123,8 @@ isFavoriteByDefault= $true `
   isFavoriteByDefault= $false `
 }
 ```
+> [!Note]
+> It can take up to 24 hours for teams users to see a custom template change in the gallery.

```

Comment:
```suggestion
> It can take up to 24 hours for Teams users to see a custom template change in the gallery.

```

Reply:
Do you mean Teams (the app) or the teams of Teams here? 😄 Thanks @JasonLewisMSFT

Reply:
It could be either, as we are talking about create/updating teams.  I copied this note from another page that already had the disclaimer.   Do you want me to change to "T"?

Reply:
Probably @JohanFreelancer9 knows better, thanks

Reply:
As we are talking about Teams users and not the teams as such, I would suggest changing it to "T". Thanks.